react-app/src/utils/formatters.tsx

```diff
@@ -0,0 +1,27 @@
+import { Chip } from '@mui/material';
+import React from 'react';
+
+export const dateFormatter = (input: any) => {
+  if (!(input instanceof Date)) {
+    input = new Date(input);
+  }
+  return `${input.getDay()}/${input.getMonth() + 1}/${input.getFullYear()}`;
```

Comment:
Based on the screenshots, I think we're looking for a MMM DD, YYYY format. 
![image](https://github.com/bcgov/PIMS/assets/37922247/fa4689c5-8dac-4f95-881b-75b7e96905af)